Fix ``dump_kwargs`` `TypeError` caused by migration to Pydantic 2.
